### ISSUE TITLE
cornell_box example throws error in 0.6.8

### ISSUE THREAD

**Issue by FanShupei** [taichi-dev/taichi#1190]
**Describe the bug**

`ti example cornell_box` throws error in version 0.6.8
I confirm cornell_box works at version 0.6.7 

**To Reproduce**
Run `ti example cornell_box`

My platform version is:
Taichi: **0.6.8** at PyPI
Python: 3.7.5
OS: Windows 10

**Log/Screenshots**

Exception Traceback:

```
[Taichi] mode=release
[Taichi] version 0.6.8, supported archs: [cpu, cuda, opengl], commit 9f4fd27c, python 3.7.5

*******************************************
**      Taichi Programming Language      **
*******************************************
Running example cornell_box ...
Traceback (most recent call last):
  File "C:\Program Files (x86)\Microsoft Visual Studio\Shared\Python37_64\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "C:\Program Files (x86)\Microsoft Visual Studio\Shared\Python37_64\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "D:\Fsp\py-env\taichi\Scripts\ti.exe\__main__.py", line 9, in <module>
  File "d:\fsp\py-env\taichi\lib\site-packages\taichi\main.py", line 876, in main
    return cli()
  File "d:\fsp\py-env\taichi\lib\site-packages\taichi\main.py", line 27, in wrapper
    result = func(*args, **kwargs)
  File "d:\fsp\py-env\taichi\lib\site-packages\taichi\main.py", line 95, in __call__
    return getattr(self, args.command)(sys.argv[2:])
  File "d:\fsp\py-env\taichi\lib\site-packages\taichi\main.py", line 165, in example
    runpy.run_path(target, run_name='__main__')
  File "C:\Program Files (x86)\Microsoft Visual Studio\Shared\Python37_64\lib\runpy.py", line 263, in run_path
    pkg_name=pkg_name, script_name=fname)
  File "C:\Program Files (x86)\Microsoft Visual Studio\Shared\Python37_64\lib\runpy.py", line 96, in _run_module_code
    mod_name, mod_spec, pkg_name, script_name)
  File "C:\Program Files (x86)\Microsoft Visual Studio\Shared\Python37_64\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "D:\Fsp\py-env\taichi\Lib\site-packages\taichi\examples\cornell_box.py", line 450, in <module>
    render()
  File "d:\fsp\py-env\taichi\lib\site-packages\taichi\lang\kernel.py", line 530, in wrapped
    return primal(*args, **kwargs)
  File "d:\fsp\py-env\taichi\lib\site-packages\taichi\lang\kernel.py", line 460, in __call__
    self.materialize(key=key, args=args, arg_features=arg_features)
  File "d:\fsp\py-env\taichi\lib\site-packages\taichi\lang\kernel.py", line 340, in materialize
    taichi_kernel = taichi_kernel.define(taichi_ast_generator)
  File "d:\fsp\py-env\taichi\lib\site-packages\taichi\lang\kernel.py", line 337, in taichi_ast_generator
    compiled()
  File "D:\Fsp\py-env\taichi\Lib\site-packages\taichi\examples\cornell_box.py", line 433, in render
    hit_pos, hit_normal, hit_color)
  File "d:\fsp\py-env\taichi\lib\site-packages\taichi\lang\kernel.py", line 55, in __call__
    ret = self.compiled(*args)
  File "D:\Fsp\py-env\taichi\Lib\site-packages\taichi\examples\cornell_box.py", line 338, in sample_direct_light
    light_pdf = compute_area_light_pdf(hit_pos, to_light_dir)
  File "d:\fsp\py-env\taichi\lib\site-packages\taichi\lang\kernel.py", line 55, in __call__
    ret = self.compiled(*args)
  File "D:\Fsp\py-env\taichi\Lib\site-packages\taichi\examples\cornell_box.py", line 254, in compute_area_light_pdf
    l_cos = light_normal.dot(-ray_dir)
  File "d:\fsp\py-env\taichi\lib\site-packages\taichi\lang\util.py", line 189, in wrapped
    return func(*args, **kwargs)
  File "d:\fsp\py-env\taichi\lib\site-packages\taichi\lang\matrix.py", line 751, in dot
    return (self.transpose() @ other).subscript(0, 0)
  File "d:\fsp\py-env\taichi\lib\site-packages\taichi\lang\matrix.py", line 186, in __matmul__
    return self.make_from_numpy(self.to_numpy() @ other.to_numpy())
  File "d:\fsp\py-env\taichi\lib\site-packages\taichi\lang\util.py", line 201, in wrapped
    f'{func.__name__} cannot be called in Taichi-scope'
AssertionError: to_numpy cannot be called in Taichi-scope
```

**Comment by BlurryLight** [comment]
An related fix is here https://github.com/taichi-dev/taichi/pull/1184. 
Before it is merged, just downgrade taichi to 0.6.7

**Comment by FanShupei** [comment]
I update taichi to 0.6.10 and confirm it's solved, so I will close the issue.
